c2corg_ui/static/js/editing/routeediting.js

```diff
@@ -34,6 +34,13 @@ app.RouteEditingController = function($scope, $element, $attrs, $http,
     $uibModal, $compile, appLang, appAuthentication, ngeoLocation, appAlerts,
     appApi, authUrl, appDocument, appUrl, imageUrl) {
 
+
+  /**
+   * @type {angular.$compile}
+   * @private
+   */
+  this.compile__ = $compile;
```

Comment:
Couldn't you use the ``compile_`` property of the parent class: https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/editing/documentediting.js#L174

Then you probably have to make the property ``@public`` and rename it to ``this.compile``.